halo/registry/keeper/keeper.go

```diff
@@ -51,15 +48,14 @@ func NewKeeper(
 	}
 
 	address := common.HexToAddress(predeploys.PortalRegistry)
-	protalReg, err := bindings.NewPortalRegistryFilterer(address, ethCl)
+	protalReg, err := bindings.NewPortalRegistryFilterer(address, nil) // Passing nil backend if safe since only Parse functions are used.
```

Comment:
```suggestion
	protalReg, err := bindings.NewPortalRegistryFilterer(address, nil) // Passing nil backend is safe since only Parse functions are used.
```